client/package.json

```diff
@@ -8,6 +8,7 @@
     "@material-ui/icons": "^4.4.3",
     "@material-ui/pickers": "^3.2.6",
     "dayjs": "^1.8.16",
+    "moment": "^2.24.0",
```

Comment:
Hmm, I already installed dayjs in this project as a more lightweight moment alternative, and that's what's being used in FundraiserCreate. Please tag me on slack about this. We will _have to_ choose one or the other.

Reply:
I used moment.js in the details page just for format conversion as it appears to process inputs seamlessly. We can decide based on requirements on slack